Source/Core/Constants/Constants+LocalizationKeys.swift

```diff
@@ -1,109 +0,0 @@
-//
-//  Constants+LocalizationKeys.swift
-//  Frames
-//
-//  Copyright © 2022 Checkout. All rights reserved.
-//
-
-extension Constants {
-
-    struct LocalizationKeys {
-
-        static let optionalInput = "OptionalInputField".localized
-
-        struct PaymentForm {
-          struct Header {
-            static let title = "PaymentHeaderTitle".localized
-            static let subtitle = "PaymentHeaderSubtitle".localized
-
-          }
-
-            struct Cardholder {
-                static let title = "CardholderInputTitle".localized
-            }
```

Comment:
Was this file being refactored part of the automatic linting? We've lost a lot of detail in the process, as the naming is very flat and missing the context that this structure was adding.